secrets/certregistry/certregistry.go

```diff
@@ -13,7 +13,7 @@ import (
 // ensuring synchronized access to them.
 type CertRegistry struct {
 	lookup map[string]*tls.Certificate
-	mx     *sync.Mutex
+	mx     sync.Mutex
```

Comment:
Lets move it above the lookup - I think there is a convention to put mutex above the fields it guards.
Also maybe rename `mx` to `mu`:
```
$ grep -o -I -E '[a-zA-Z0-9]+\s+sync.Mutex' -r | cut -d: -f2 | cut -d' ' -f1 | sort | uniq -c
      1 closeMx
      1 m
     16 mu
      3 mx
```

Reply:
will add another PR

Reply:
Did not notice your comment before I created https://github.com/zalando/skipper/pull/2820